src/qt/modaloverlay.h

```diff
@@ -9,7 +9,7 @@
 #include <QWidget>
 
 //! The required delta of headers to the estimated number of available headers until we show the IBD progress
-static const int REQ_HEADER_HEIGHT_DELTA_SYNC = 24;
+static constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
```

Comment:
Maybe `Assume we are syncing headers, should the estimated number of missing headers be larger than that number`?